### DISCUSSION TITLE
How to build a Windows release, which includes all Qt dependencies?

### DISCUSSION THREAD

**Discussion by ghackenberg** [leozide/leocad#868] [Q&A] [answered]
Hi @leozide,

I would like to **build a Windows release** of my LeoCAD [fork/branch](https://github.com/ghackenberg/leocad/tree/hub), which I can distribute to colleagues and students.

I managed to get **QtCreator** up an running for LeoCAD development ... **that works perfectly fine**!

But I have troubles configuring my build system, such that a distribuable release with all the Qt dependencies comes out. When I run the release build from outside QtCreator, the executable tries to load the Qt dependencies dynamically, but cannot find them on my system obviously.

![image](https://github.com/leozide/leocad/assets/7525462/78d885ee-cd0d-4e90-862b-07dfce937723)

Of course I could add the Qt framework binaries to my `PATH` environment variable, but that would not work on my colleagues and my students computers. I tried adding QMake configuration options instead as suggested [here](https://stackoverflow.com/questions/20799161/how-to-make-qt-and-qtcreator-link-the-libraries-statically-instead-of-dynamic) and shown in the following screenshot. But that did not work...

![image](https://github.com/leozide/leocad/assets/7525462/55614474-abcc-4e4f-8a54-de8bf11250d1)

Can you help me on this? That would be great! Thank you very much in advance!

Best regards, Georg

**Comment by leozide** [comment, marked as answer]
Use windeployqt to copy the dlls

**Reply by ghackenberg** [reply]
Dear @leozide,

thank you very much for the quick answer. Now it works!

BR, Georg
